deploy/bit-builder-recipes/f1.yaml

```diff
@@ -0,0 +1,20 @@
+# Build-time bitbuilder design configuration for the FireSim Simulation Manager
+# See https://docs.fires.im/en/stable/Advanced-Usage/Manager/Manager-Configuration-Files.html for documentation of all of these params.
+
+###########
+# Schema:
+###########
+# # Class name of the bitbuilder type.
+# # This can be determined from `deploy/buildtools/bitbuilder.py`).
+# bitbuilder_type: <TYPE NAME>
+# args:
+#     # Bitbuilder arguments that are passed to the `BitBuilder`
+#     # object. Determined by looking at `_parse_args` function of class.
+#     <K/V pairs of args>
+
+bit_builder_type: F1BitBuilder
+args:
+    # REQUIRED: name of s3 bucket
+    s3_bucket_name: firesim
+    # REQUIRED: append aws username to s3 bucket?
+    append_userid: true
```

Comment:
Can you make this "append_userid_region" instead? And append both userid and region to the base bucket name.

There have been issues in the past with buckets created in other regions.

Reply:
Sure.